tests/Support/SupportMacroableTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
```

Comment:
NOTE: I think this is one where we pulled in Macroable as a dependency but failed to pull in its test; so it makes sense for this to be here.